src/picosvg/svg_transform.py

```diff
@@ -171,8 +171,7 @@ def inverse(self):
         a, b, c, d, e, f = self
         det = self.determinant()
         a, b, c, d = d / det, -b / det, -c / det, a / det
-        e = -a * e - c * f
-        f = -b * e - d * f
+        e, f = -a * e - c * f, -b * e - d * f
```

Comment:
for me at least putting these on the same lines is materially reducing legibility and making it harder to see what the diff is